pkg/client/clientset/versioned/clientset.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC All Rights Reserved.
+// Copyright 2023 Google LLC All Rights Reserved.
```

Comment:
Wondering why the CRD files for updated? This is a gRPC change, yeah?

Reply:
I'll delete the generated files and see if it complains.